dll/apisets/api-ms-win-core-file-ansi-l2-1-0.spec

```diff
@@ -0,0 +1,5 @@
+
+# This file is autogenerated by update.py
+
+@ stdcall CopyFileExA() kernel32.CopyFileExA
```

Comment:
From my point of view, all (new) apisets should have a `-version=`, at least `0x600+`...

Reply:
I could verify when the functions were introduced from msdn and add the version tags there.

Reply:
Why should they have a version?
on 5.2 there are simply no apisets, so there is no need to hide anything